app/javascript/onboarding/components/Navigation.jsx

```diff
@@ -42,7 +42,15 @@ class Navigation extends Component {
         >
           {!hidePrev && (
             <button onClick={prev} className="back-button" type="button">
-              Back
+              <svg
+                width="16"
+                height="16"
+                fill="none"
+                className="crayons-icon"
+                xmlns="http://www.w3.org/2000/svg"
+              >
+                <path d="M3.828 7H16v2H3.828l5.364 5.364-1.414 1.414L0 8 7.778.222l1.414 1.414L3.828 7z" />
+              </svg>
```

Comment:
I feel like I'm the one who is always asking other people in PR reviews, _"Did you optimize this svg?"_, so...before anyone asks me that, let me just say that yes, I have optimized this 😛

Reply:
lol well oops I haven't done that at all 😂 TIL about `svgo` tho!

Reply:
😂 i've reduced svg's by up to 60% and the day i did that for the first time, i was sold on svgo. 

i think it can be helpful when you have a ton of svgs rendering on the page (which, tbqh, we don't have for onboarding) and you need to load them all

Reply:
@vaidehijoshi I have a draft PR for svgo https://github.com/thepracticaldev/dev.to/pull/7020 - I'll try to refresh it and get it to completion soon. I was blocked by some test failures